File: Praktikumsverwaltung_AndroidApp/app/src/main/java/com/obercoder/praktikumsverwaltung_androidapp/pkgData/Company.java
package com.obercoder.praktikumsverwaltung_androidapp.pkgData;

import org.bson.types.ObjectId;

/**
 * Created by Sasa on 17.11.2017.
 */

public class Company {
    private ObjectId id;
    private String name;
    private String location;
    private int numberOfEmployees;
    private String contactPerson;


    public Company(ObjectId id, String name, String location, int numberOfEmployees, String contactPerson) {
        this.id = id;
        this.name = name;
        this.location = location;
        this.numberOfEmployees = numberOfEmployees;
        this.contactPerson = contactPerson;
    }

    public ObjectId getId() {
        return id;
    }

    public void setId(ObjectId id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public int getNumberOfEmployees() {
        return numberOfEmployees;
    }

    public void setNumberOfEmployees(int numberOfEmployees) {
        this.numberOfEmployees = numberOfEmployees;
    }

    public String getContactPerson() {
        return contactPerson;
    }

    public void setContactPerson(String contactPerson) {
        this.contactPerson = contactPerson;
    }
}

File: DesktopApp/Praktikumsverwaltung_DesktopApp/Praktikumsverwaltung_DesktopApp/pkgData/Company.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Praktikumsverwaltung_DesktopApp.pkgData
{
    class Company
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public int NumberOfEmployees { get; set; }
        public string ContactPerson { get; set; }

        public Company(string id, string name, string location, int numberOfEmployees, string contactPerson)
        {
            Id = id;
            Name = name;
            Location = location;
            NumberOfEmployees = numberOfEmployees;
            ContactPerson = contactPerson;
        }
    }
}
